shortlist/solutions/C1.md

```diff
@@ -0,0 +1,16 @@
+## Proof
+
+We tile the rectangle as so:
+[Figure 1] (where the corners are black)
+
+Consequently, there is at least 1 rectangle with odd side length and black corners. (**Lemma 1**)
```

Comment:
I would consider outlining the solution in the style of Evan Chan as the opening, and then diving in on details as time will allow.

I can imagine something like:

We can tile the rectangle as a checkerboard with black squares on the corners and prove that the partition must contain a rectangle that has 4 black corners, which will then satisfy the requirement that its distances to all 4 sides are all odd or all even. The proof will hinge upon the fact that this checkerboard has k white squares and k+1 black squares for some value of k, and that only rectangles with all-black corners satisfy the requirement that there are more black squares than white.